_locales/en/messages.json

```diff
@@ -267,15 +267,23 @@
   },
   "options_option_timeInactive_minutes": {
```

Comment:
Let's stick with just `minutes` and `seconds` and let minutes increase to 43,200 (30 days worth of seconds). The UI could update to show different inputs, but the value will ultimately be stored as a number of minutes.

In hindsight, it was not needed to store both `minutes` and `seconds`. The value could have remained in seconds and displayed differently in the UI.